internal/controller/database_secret_handler.go

```diff
@@ -140,7 +200,7 @@ func isObjectUpdated(e event.UpdateEvent) bool {
 	if isSecret {
 		// only annotated secrets are watched
 		annotations := secretNew.ObjectMeta.GetAnnotations()
-		dbcrName, ok := annotations[DbSecretAnnotation]
+		dbcrName, ok := annotations[consts.CREATOR_ANNOTATION_KEY]
```

Comment:
label: `used-by`

Reply:
should be fixed

Reply:
Was thinking also this would not annotation and instead label. Label would make the object able to be filtered/selected. Think a "used-by" is a good case for this.

Reply:
But am I not already not using annotations. I guess the code snippet here is outdated. But I'll check again

Reply:
Ouch, I'm not, I'll fix